.github/workflows/ci.yml

```diff
@@ -108,6 +110,43 @@ jobs:
           cache-from: type=registry,ref=${{ env.DOCKER_IMAGE_TAG }}
           cache-to: type=inline
 
+  tag-publish:
+    if: (github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/'))
+    needs: [ prettier, es-lint, tests-finish ]
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_CACHE_SOURCE: staging
+      IMAGE_REGISTRY: safeglobal
+      IMAGE_NAME: safe-client-gateway-nest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Extract commit metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAME }}
+      - run: |
+          BUILD_NUMBER=${{ github.sha }}
+          echo "BUILD_NUMBER=${BUILD_NUMBER::7}" >> "$GITHUB_ENV"
+      - uses: docker/setup-qemu-action@v2.2.0
+        with:
+          platforms: arm64
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - uses: docker/build-push-action@v4
+        with:
+          platforms: linux/amd64,linux/arm64
```

Comment:
I took this from the `staging` image build process. AFAIK we could omit `linux/amd64` and rely on `linux/arm64` only, but I think the build is made in parallel removing `linux/amd64` won't decrease the build time. I left both architectures as targets but no strong opinion on this subject.